src/EnergyPlus/OutAirNodeManager.cc

```diff
@@ -390,19 +439,29 @@ namespace OutAirNodeManager {
 				Node( NodeNum ).OutAirDryBulb = OutDryBulbTempAt( Node( NodeNum ).Height );
 				Node( NodeNum ).OutAirWetBulb = OutWetBulbTempAt( Node( NodeNum ).Height );
 			}
+			Node( NodeNum ).OutAirWindSpeed = WindSpeed;
+			Node( NodeNum ).OutAirWindDir = WindDir;
 
-			if ( Node( NodeNum ).EMSOverrideOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = Node( NodeNum ).EMSValueForOutAirDryBulb;
+			// Set node data to local air node values if defined
+			if ( Node( NodeNum ).SchedOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirDryBulbSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWetBulb ) Node( NodeNum ).OutAirWetBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirWetBulbSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWindSpeed ) Node( NodeNum ).OutAirWindSpeed = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindSpeedSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWindDir ) Node( NodeNum ).OutAirWindDir = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindDirSchedNum );
 
-			if ( Node( NodeNum ).EMSOverrideOutAirWetBulb ) {
-				Node( NodeNum ).OutAirWetBulb = Node( NodeNum ).EMSValueForOutAirWetBulb;
-				Node( NodeNum ).HumRat = PsyWFnTdbTwbPb( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).OutAirWetBulb, OutBaroPress );
-				Node( NodeNum ).Enthalpy = PsyHFnTdbW( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).HumRat );
-			} else {
-				Node( NodeNum ).HumRat = OutHumRat;
-				Node( NodeNum ).Enthalpy = PsyHFnTdbW( Node( NodeNum ).OutAirDryBulb, OutHumRat );
-			}
+			// Set node data to EMS overwritten values if defined
+			if ( Node( NodeNum ).EMSOverrideOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = Node( NodeNum ).EMSValueForOutAirDryBulb;
+			if ( Node( NodeNum ).EMSOverrideOutAirWetBulb ) Node( NodeNum ).OutAirWetBulb = Node( NodeNum ).EMSValueForOutAirWetBulb;
+			if ( Node( NodeNum ).EMSOverrideOutAirWindSpeed ) Node( NodeNum ).OutAirWindSpeed = Node( NodeNum ).EMSValueForOutAirWindSpeed;
+			if ( Node( NodeNum ).EMSOverrideOutAirWindDir ) Node( NodeNum ).OutAirWindDir = Node( NodeNum ).EMSValueForOutAirWindDir;
 
 			Node( NodeNum ).Temp = Node( NodeNum ).OutAirDryBulb;
+			if ( Node( NodeNum ).SchedOutAirDryBulb || Node( NodeNum ).SchedOutAirWetBulb ) {
+				Node( NodeNum ).HumRat = PsyWFnTdbTwbPb( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).OutAirWetBulb, OutBaroPress );				
```

Comment:
What happens if the user schedules a dry bulb temperature that is invalid with the wet bulb temperature provided by the weather file.  Or if the user just schedules both and the combination is invalid?  Where is the error handling for that?  I may find it as I keep reviewing, but I don't think I've seen it so far.